Blimp/system.cpp

```diff
@@ -262,7 +262,7 @@ void Blimp::allocate_motors(void)
         break;
     }
     if (motors == nullptr) {
-        AP_HAL::panic("Unable to allocate FRAME_CLASS=%u", (unsigned)g2.frame_class.get());
+        AP_BoardConfig::config_error("Unable to allocate FRAME_CLASS=%u", (unsigned)g2.frame_class.get());
```

Comment:
```suggestion
        AP_BoardConfig::allocation_error("Unable to allocate FRAME_CLASS=%u", (unsigned)g2.frame_class.get());
```

Reply:
I am following the pattern followed in other vehicles for same situation, shall I change it for others as well?